pages/docs/data-ops/dbt-integration.mdx

```diff
@@ -1,33 +1,112 @@
+import { Steps } from 'nextra-theme-docs'
+
 # dbt Integration
 
 ## Overview
 
-Glean can import models directly from dbt. This is useful for incorporating Glean into existing DataOps workflows, allowing you to make changes to your dbt schema while identifying issues in downstream visualizations and dashboards.
+Glean can import models directly from dbt by specifying metrics in your dbt yaml file. Glean will automatically sync dbt documentation and keep metrics in sync with downstream dashboards (whether they are checked into code or not).  It's also possible to integrate with CI/CD processes.
 
-You must add the following fields to the `meta` key of each dbt model you want imported to Glean:
+<Steps>
 
-```yaml
-# schema.yml
+### Configure your `dbt_project.yml`
+
+You can use the `glean-defaults` key to specify the glean version (which is required) and connection information.
+
+```yaml filename="dbt_project.yml" copy
+
+models:
+  +meta:
+    # glean-defaults settings will be merged into each Glean model
+    glean-defaults:
+      glean: "1.0"
+      # preventUpdatesFromUI defaults to false, but can be set to true
+      preventUpdatesFromUI: false
+      source:
+        connectionName: Uploads # the name of the conneciton in Glean
+        # can use jinja
+        # schema is always optional
+        schema: "{{ env_var('DBT_SCHEMA') }}"
```

Comment:
would maybe suggest using `"{{ target.schema }}"` which might be more familiar to dbt users